### PR TITLE
[DONE] Add some improvements for quizzes

### PR DESCRIPTION
**Pull request by AymericJak**
Before sending your pull request, make sure the following are done:

* [x] You have read our [contribution guidelines](https://github.com/EsupPortail/Esup-Pod/blob/master/CONTRIBUTING.md).
* [x] Your PR targets the `develop` branch.
* [x] The title of your PR starts with `[WIP]` or `[DONE]`.

### CODE REVIEW

File: pod/quiz/templatetags/video_quiz.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ def is_quiz_accessible(context: dict, video: Video) -> bool:
     quiz = get_video_quiz(video)
     if quiz:
         request = context["request"]
+        if quiz.is_draft:
+            return video.owner == request.user
```
Comment (ptitloup) [inline pod/quiz/templatetags/video_quiz.py:28]: je rajouterai un `or request.usr.is_superuser`

**Review by ptitloup — APPROVED** [review]
une petite améioration mais sinon ca me semble ok

File: pod/quiz/templatetags/video_quiz.py (hunk 1 above)
Comment (AymericJak) [reply pod/quiz/templatetags/video_quiz.py]: Fait dans le commit https://github.com/EsupPortail/Esup-Pod/pull/1155/commits/4cd8fef49beb1a83a23752d2758b37d52f707b47

File: pod/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3058,8 +3056,8 @@ msgid ""
 "a>."
 msgstr ""
```
Comment (gcondess) [inline pod/locale/fr/LC_MESSAGES/django.po:3057]: I'm not fan of strings begin with empty "" (eg msgid "" or msgstr "") but as they are so much, this seems to be the rule !

**Review by gcondess — APPROVED** [review]
RAS (unless code formating for language code with long string, but it's just a personnal preference ;)

**Review by SebastienCozeDev — APPROVED** [review]
Tout est ok pour moi ! Merci pour cette contribution !

File: pod/locale/fr/LC_MESSAGES/django.po (hunk 2 above)
Comment (AymericJak) [reply pod/locale/fr/LC_MESSAGES/django.po:3057]: It's automatically generated when executing `make lang`.

So, it's normal :)

**Pull request merged**